draft-ietf-cbor-7049bis.md

```diff
@@ -1318,11 +1318,10 @@ application, directly using the numbers for the keys is useful.
 If multiple types of keys are to be used, consideration should be
 given to how these types would be represented in the specific
 programming environments that are to be used.  For example, in
-JavaScript objects, a key of integer 1 cannot be distinguished from a
```

Comment:
This is confusing to people who do not know about JavaScript Maps and are only aware of doing this as JSON.  A reference should probably be included.

Reply:
Done.